src/WorkerFarm.js

```diff
@@ -97,6 +99,9 @@ for (let key in EventEmitter.prototype) {
 }
 
 function getNumWorkers() {
+  if (process.env.PARCEL_WORKERS) {
+    return parseInt(process.env.PARCEL_WORKERS);
```

Comment:
I recommend you to specify the `radix` argument, else it'll be infered. Eg. `parseInt('0x0101') = 257` while `parseInt('0x0101', 10) = 0`.